src/lib/onboarding/router.ts

```diff
@@ -99,11 +103,12 @@ router.post('/submit-verified', async (req, res) => {
   }
 });
 
-router.post('/waiting-list', parser.urlencoded(), async (req, res) => {
+router.post('/waiting-list', async (req, res) => {
   try {
     const account = req.body;
 
-    if (!validateAccount(account)) throw new AccountNonValidError(account);
+    if (!validateAccountWithResponse(account))
```

Comment:
What happened to `isValidAccountWithResponse` naming scheme?

Reply:
it has been fixed in a parallel dimension https://github.com/paraswap/paraswap-volume-tracker/pull/47/files#diff-872aa1bf3a747dd06dfa857ac1f59b700aa7d8039905eaa4a6e70ac9bff5a7c1R111